filters/resize_test.go

```diff
@@ -17,14 +17,40 @@ func TestResize_Name(t *testing.T) {
 	assert.Equal(t, "resize", r.Name())
 }
 
-func TestResize_CreateOptions(t *testing.T) {
```

Comment:
You should not delete this test, when `keepAspectRatio` is not specified, you should test that it defaults to false, and that the aspect ratio is not preserved.